junit-platform-gradle-plugin/src/main/groovy/org/junit/platform/gradle/plugin/FiltersExtension.groovy

```diff
@@ -51,4 +65,21 @@ class FiltersExtension {
 		includeClassNamePatterns.addAll(patterns)
 	}
 
+	/**
+	 * Add a pattern to the list of <em>excluded</em> patterns.
+	 */
+	void excludeClassNamePattern(String pattern) {
+		excludeClassNamePatterns(pattern)
+	}
+
+	/**
+	 * Add patterns to the list of <em>included</em> patterns.
```

Comment:
Though... TBH, you did a great job of finding all other occurrences of _included_ and replacing them. 👍